content/en/moved.html

```diff
@@ -0,0 +1,41 @@
++++
+title = "Voter Registration After a Move"
+translationKey = "about"
```

Comment:
Please update the translationkey to be unique to this voter guide. I suggest using `recently_moved`